test/integration/targets/eos_static_routes/tasks/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
- "rtt.yaml" test is missing
- All IP addresses should be to documentation IP range in https://tools.ietf.org/html/rfc5737 and https://tools.ietf.org/html/rfc3849, you can use something like https://www.site24x7.com/tools/ipv4-subnetcalculator.html for subnetting the allocated blocks, if required.